tests/token/conftest.py

```diff
@@ -0,0 +1,130 @@
+
+from spylib.token import (
+    OfflineTokenABC,
+    OfflineTokenResponse,
+    OnlineTokenABC,
+    OnlineTokenResponse,
+)
+
+test_database: Dict[str, Dict] = {
+    "online": {},
+    "offline": {},
+}
+
+
+# These are the fixtures for the subdirectories
+
+
+@pytest.fixture(scope='session', autouse=True)
+def database():
+    return test_database
+
+
+@pytest.fixture(scope='session', autouse=True)
+def OnlineToken():
+    global test_database
+    test_database = {"offline": {}, "online": {}}
+
+    class OnlineToken(OnlineTokenABC):
+        async def save_token(self):
+            test_database['online'][self.store_name] = {}
+            test_database['online'][self.store_name][self.associated_user.id] = self
+
+        @classmethod
+        async def load_token(cls, store_name: str, user_id: str) -> OnlineToken:
+            return test_database['online'][store_name][user_id]
+
+    yield OnlineToken
+
+    test_database = {"offline": {}, "online": {}}
+
+
+@pytest.fixture(scope='session', autouse=True)
+def OfflineToken():
+    global test_database
+    test_database = {"offline": {}, "online": {}}
+
+    class OfflineToken(OfflineTokenABC):
+        async def save_token(self):
+            test_database['offline'][self.store_name] = self
+
+        @classmethod
+        async def load_token(cls, store_name: str) -> OfflineToken:
+            return test_database['offline'][store_name]
+
+    yield OfflineToken
+
+    test_database = {"offline": {}, "online": {}}
+
+
+@pytest.fixture(scope='session', autouse=True)
+def online_token_data() -> OnlineTokenResponse:
+    return OnlineTokenResponse(
+        access_token='ONLINETOKEN',
+        scope=','.join(['write_products', 'read_customers']),
+        expires_in=86399,
+        associated_user_scope=','.join(['write_orders', 'read_products']),
+        associated_user={
+            "id": 902541635,
+            "first_name": "John",
+            "last_name": "Smith",
+            "email": "john@example.com",
+            "email_verified": True,
+            "account_owner": True,
+            "locale": "en",
+            "collaborator": False,
+        },
+    )
+
+
+@pytest.fixture(scope='session', autouse=True)
+def offline_token_data() -> OfflineTokenResponse:
+    return OfflineTokenResponse(
+        access_token='OFFLINETOKEN',
+        scope=','.join(['write_products', 'read_customers']),
+    )
+
+
+class AppInformation(BaseModel):
+    """
+    Information about the store for tests.
+    """
+
+    api_version: str = '2021-04'
+    store_name: str = "Test-Store"
+    client_id: int = 1
+    client_secret: int = 2
+    code: int = 3
+    public_domain: str = 'test.testing.com'
+    private_key: str = 'TESTPRIVATEKEY'
+
+
+@pytest.fixture(scope='session', autouse=True)
+def app_information() -> AppInformation:
+    return AppInformation()
+
+
+@pytest.fixture(scope='session', autouse=True)
+def MockHTTPResponse():
+    @dataclass
+    class MockHTTPResponse:
```

Comment:
Wait that works to have the two with the same name?
That's weird and confusing. Let's not do that please. Could be:
```suggestion
def MockHTTPResponse():
    @dataclass
    class MockHTTPResponseCls:
```